tallycash/extension
# Blue Grouse Migration 🐦 - migrate redux state to include keyringMetadata.

Now that we use keyringMetadata to store extra information about keyrings, we need a migration to add it to the redux store of existing tally users.

### How to test:

-  uninstall tally wallet from chrome (or open incognito)
- `git fetch --all`
- `git checkout e400653e13ef9cb42279a9fb3f2cafbaa5b5f495` // version 0.12
- `yarn dev`
-  Add a new tally wallet & import a wallet (or two)
- `ctrl+c` the yarn dev process
- `git checkout migrate-keyringmetadata`
- `yarn dev`
-  click the update button in chrome://extensions

After these steps are completed - the app should be useable and the accounts list should properly display your previously created/imported wallets. 

Note: The imported wallets will display as `Tally` wallets since we did not keep track of the `source` of a wallet in version 0.12
